client/packages/host/src/components/Initialise/hooks.ts

```diff
@@ -1,21 +1,20 @@
 import { useEffect, useState } from 'react';
 import { AppRoute } from '@openmsupply-client/config';
 import {
-  AuthenticationError,
-  LocaleKey,
-  initialisationStatusType,
-  TypedTFunction,
   useNavigate,
   useTranslation,
+  ErrorWithDetailsProps,
+  InitialisationStatusType,
 } from '@openmsupply-client/common';
 import { useHost } from '../../api/hooks';
+import { mapSyncError } from '../../api/api';
 
 const STATUS_POLLING_INTERVAL = 500;
 const DEFAULT_SYNC_INTERVAL_IN_SECONDS = 300;
 
 interface InitialiseForm {
   // Error on validation of sync credentials, there is another error for sync progress
-  siteCredentialsError?: AuthenticationError;
+  siteCredentialsError: ErrorWithDetailsProps | null;
```

Comment:
use null instead of undefined because `setSiteCredentialsError(null)` seemed more explanatory then `setSiteCredentialsError()`

Reply:
nice 😃